db/migrate/20150118173952_add_is_required_to_assignments.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsRequiredToAssignments < ActiveRecord::Migration
+  def change
+      add_column :assignments, :is_required, :boolean, :default => "false"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 4) spaces for indentation.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.